tensorflow/core/kernels/mkl_fused_ops_test.cc

```diff
@@ -604,9 +604,9 @@ class BiasCacheTest : public OpsTestBase {
             .Input(FakeInput(DT_FLOAT))  // Max-input
             .Input(FakeInput(DT_FLOAT))  // Min-filter
             .Input(FakeInput(DT_FLOAT))  // Max-filter
-            .Input(FakeInput(DT_FLOAT))  // min-output
+            .Input(FakeInput(DT_FLOAT))  // Min-output
             .Input(FakeInput(DT_FLOAT))  // max-output
-            .Input(FakeInput(DT_UINT8))  // MKl second tensor
+            .Input(FakeInput(DT_UINT8))  // MKL second tensor
             .Input(FakeInput(DT_UINT8))  // MKl second tensor
```

Comment:
Several comment lines below this still has `MKl`. Please replace all with `MKL`.

Reply:
Fixed.